tests/functional/tests/smoke.py

```diff
@@ -134,6 +134,7 @@ def generate_expected_plan(self, backup: bool = True) -> str:
         backup_plan = "\tBack up MySQL databases\n" if backup else ""
         return (
             "Upgrade cloud from 'ussuri' to 'victoria'\n"
+            "\tCheck application vault is not sealed\n"
```

Comment:
(non-blocking) nit: `Verify application vault is not sealed\n` just to be consistent

Reply:
Updated.